packages/editor/src/components/parts/condition/ConditionTable.tsx

```diff
@@ -44,7 +44,7 @@ const ConditionTable = ({ data, onChange }: { data: Condition[]; onChange: (chan
         header: header => <TextHeader header={header} name='Expression' />,
         cell: cell => (
           <ReorderWrapperIcon>
-            <ScriptCell cell={cell} type={IVY_SCRIPT_TYPES.BOOLEAN} browsers={['attr', 'func', 'type']} />
+            <ScriptCell cell={cell} type={IVY_SCRIPT_TYPES.BOOLEAN} browsers={['attr', 'func', 'type']} placeholder='Enter an Expression' />
```

Comment:
remove placeholder here?

Reply:
I have set this placeholder, so that there is a small hint on what two write inside this cell. But I admit, its not much of additional information 😅 
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/f7779569-3d0e-4edb-83ca-ece0cfc835dd)

Reply:
ah sorry. of course :D